src/neo4j/_codec/packstream/v1/__init__.py

```diff
@@ -16,16 +16,55 @@
 # limitations under the License.
 
 
+import typing as t
 from codecs import decode
 from contextlib import contextmanager
 from struct import (
     pack as struct_pack,
     unpack as struct_unpack,
 )
 
+from ...hydration import DehydrationHooks
 from .._common import Structure
 
 
+NONE_VALUES: t.Tuple = (None,)
+TRUE_VALUES: t.Tuple = (True,)
+FALSE_VALUES: t.Tuple = (False,)
+INT_TYPES: t.Tuple[t.Type, ...] = (int,)
+FLOAT_TYPES: t.Tuple[t.Type, ...] = (float,)
+# we can't put tuple here because spatial types subclass tuple,
+# and we don't want to treat them as sequences
+SEQUENCE_TYPES: t.Tuple[t.Type, ...] = (list,)
+MAPPING_TYPES: t.Tuple[t.Type, ...] = (dict,)
+BYTES_TYPES: t.Tuple[t.Type, ...] = (bytes, bytearray)
+
+
+try:
+    import numpy as np
+
+    TRUE_VALUES += (np.bool_(True),)
```

Comment:
Readability wise it might be better if all of the lines used the same way of extending the list. 
`TRUE_VALUES = (*TRUE_VALUES, np.bool_(True))`